src/IO/H5/Python/CombineH5.py

```diff
@@ -0,0 +1,51 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+
+import click
+import rich
+
+import spectre.IO.H5 as spectre_h5
+
+
+@click.command(name="combine-h5")
+@click.option(
```

Comment:
Please add a comment that this needs to be changed to a list of files. I'm hesitant merging this because it's inconsistent with the other CLI commands, and with the guidelines written here: https://spectre-code.org/spectre_writing_python_bindings.html#autotoc_md148
I'd be fine with doing this in a follow-up PR.